src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1566,7 +2001,12 @@ subroutine gsi_fv3ncdf_read(grd_ionouv,cstate_nouv,filenamein,fv3filenamegin)
     endif
     do ilevtot=kbgn,kend
       vgsiname=grd_ionouv%names(1,ilevtot)
+      ! write(6,*)' gsi_fv3ncdf_read_wang1: ',ilevtot,vgsiname
```

Comment:
If commented out write won't be used in the future, please remove commented line.

Reply:
Please remove debug write statement.